db/migrations/versions/005_create_asmt_store_triggers.py

```diff
@@ -5,13 +5,14 @@
 Create Date: 2024-12-02 13:47:41.220406
 
 """
+
 from alembic import op
 from alembic_utils.pg_function import PGFunction
 from alembic_utils.pg_trigger import PGTrigger
 
 # revision identifiers, used by Alembic.
-revision = '005_create_asmt_store_triggers'
-down_revision = '004_create_assessment_store'
+revision = "005_create_asmt_store_triggers"
```

Comment:
Is this required change ?

Reply:
No. But it was the git action pre-commit that did them. So please ignore them.

Reply:
Sorted!